build-release.sh

```diff
@@ -359,7 +365,22 @@ if [ "${build_mono}" == "1" ]; then
 
   ## iOS (Mono) ##
 
-  # Not supported yet.
+  rm -rf ios_xcode
+  cp -r git/misc/dist/ios_xcode ios_xcode
+  for suffix in "" "_arkit_module" "_camera_module"; do
+    cp out/ios/templates-mono/libgodot${suffix}.iphone.opt.fat ios_xcode/libgodot${suffix}.iphone.release.fat.a
+    cp out/ios/templates-mono/libgodot${suffix}.iphone.opt.debug.fat ios_xcode/libgodot${suffix}.iphone.debug.fat.a
+  done
+  chmod +x ios_xcode/libgodot*.iphone.*
+  cd ios_xcode
+  zip -q -9 -r "${templatesdir}/iphone.zip" *
```

Comment:
I think this is why `iphone.zip` was missing from the test build:

```suggestion
  zip -q -9 -r "${templatesdir_mono}/iphone.zip" *
```

Reply:
Thanks, fixed.